ksqldb-rest-client/src/main/java/io/confluent/ksql/rest/client/KsqlRestClient.java

```diff
@@ -208,7 +208,7 @@ public void setServerAddress(final String serverAddress) {
   ) {
     KsqlTarget target = target();
     final Map<String, Object> requestPropertiesToSend = new HashMap<>(requestProperties);
-    if (ConsistencyOffsetVector.isConsistencyVectorEnabled(localProperties.toMap())) {
+    if (ConsistencyOffsetVector.isConsistencyVectorEnabled(requestPropertiesToSend)) {
```

Comment:
I made this change to have `makeQueryRequest` and `makeQueryRequestStreamed` behave similarly, i.e. both use `requestProperties`